src/tests/suite/daos_rebuild.c

```diff
@@ -1846,10 +1848,28 @@ multi_pools_rebuild_concurrently(void **state)
 static int
 rebuild_sub_setup(void **state)
 {
+	if (state != NULL && *state != NULL) {
+		save_arg = *state;
+		*state = NULL;
+	}
```

Comment:
If we can re-order these subtest case to make all cases, which share the same states, are stick together, do we still need this?

Reply:
no but this is intended to safely handles all cases...